examples/models/question_answering/Question_Answering.py

```diff
@@ -0,0 +1,484 @@
+                        title = df_covid_only['title'][paper_index] if not pd.isna(
+                            df_covid_only['title'][paper_index]) else "not available"
+                        start, end = findStartEndIndexSubstring(paragraph, answer)
+                        answer_parag = getSingleContext(paragraph, start, end)
+                        paper_citations_count, journal_api = getAPIInformations(df_covid_only['paper_id'][paper_index])
+                        journal = journal_api if journal_api else journal
+                        journal_row = scimago_jr[scimago_jr['Title'].apply(lambda x: x.lower()) == journal.lower()]
+                        journal_score = journal_row[
+                            'SJR'].item() if not journal_row.empty else "Information not available"
+                        paper_answer = {
+                            "answer": answer_parag,
+                            "title": title,
+                            "journal": journal,
+                            "author": author,
+                            "journal_score": journal_score,
+                            "paper_citations_count": paper_citations_count,
+                        }
+                        answers_list.append(paper_answer)
+                dict_queries[query] = answers_list
+            dict_task_quest['queries'] = dict_queries
+            dict_task_quest['task_name'] = data[task]['area']
+            final_dict[task] = dict_task_quest
+
+        full_code = final_dict
+
+        cnt = 1
+        task_cnt = 1
+        for task_key in full_code.keys():
+            results_table = pd.DataFrame(columns=['Question', 'Title', 'Authors', 'Answer', 'Journal', 'Journal score',
+                                                  'Paper citations count', ])
+            for question_key in full_code[task_key]['queries'].keys():
+                for idx in range(len(full_code[task_key]['queries'][question_key])):
+                    row = [question_key,
+                           full_code[task_key]['queries'][question_key][idx]['title'],
+                           full_code[task_key]['queries'][question_key][idx]['author'],
+                           BeautifulSoup(full_code[task_key]['queries'][question_key][idx]['answer']).text,
+                           full_code[task_key]['queries'][question_key][idx]['journal'],
+                           full_code[task_key]['queries'][question_key][idx]['journal_score'],
+                           full_code[task_key]['queries'][question_key][idx]['paper_citations_count'],
+                           ]
+                    results_table.loc[cnt] = row
+                    cnt += 1
+            # save as output
+            results_table.to_csv("test/task_{}_results.tsv".format(task_cnt))
+            task_cnt += 1
+        print(results_table, task_cnt)
+        return results_table
+
+    def evaluate(self, dataset_path):
+        return float(1)
+
+    def predict(self, queries=None): 
```

Comment:
the predict() function receives the user provided question and returns the answer.
refer to this pic I shared before.
![WeChat Image_20200517110217](https://user-images.githubusercontent.com/3797447/82134799-4e6e7200-982e-11ea-9b25-41178954c35b.png)